python/src/otel/otel_sdk/otel-instrument

```diff
@@ -40,5 +43,68 @@ environ["OTEL_RESOURCE_ATTRIBUTES"] = "%s,%s" % (
     environ.get("OTEL_RESOURCE_ATTRIBUTES"),
 )
 
-# start the runtime with the extra options
+import logging
+
+from opentelemetry.environment_variables import OTEL_PYTHON_DISABLED_INSTRUMENTATIONS
```

Comment:
Not sure if bootstrap can find dependencies, may need to set dependency path.

Reply:
Ah thanks for your prompt, that's a good question!

I think I was too ambitious with all these changes here, this PR is really just about propagators.

I've decided to move them to a separate PR in #125 so we can discuss all the concerns there separately.